docs/integrations/app-development/opentelemetry/jfrog-artifactory-opentelemetry.md

```diff
@@ -0,0 +1,175 @@
+---
+id: jfrog-artifactory-opentelemetry
+title: JFrog Artifactory - OpenTelemetry Collector
+sidebar_label: JFrog Artifactory - OTel Collector
+description: Learn about the Sumo Logic OpenTelemetry App for JFrog Artifactory.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/app-development/jfrog-Artifactory.png')} alt="Thumbnail icon" width="80"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The Sumo Logic App for Artifactory provides insight into your [JFrog Artifactory](https://jfrog.com/artifactory/) binary repository. The App provides preconfigured Dashboards that include an Overview of your system, Traffic, Requests and Access, Download Activity, Cache Deployment Activity, and Non-Cached Deployment Activity. Artifactory logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Artifactory-OpenTelemetry/Artifactory-Schematics.png')} alt="Artifactory-Schematics" />
```

Comment:
According to our [style guide](https://help.sumologic.com/docs/contributing/style-guide/#images), "Images must be added to the static/img folders." Is it acceptable to point to an image outside of our file structure?

I have this question about all the <img src> tags in this pull request.